ansible/configs/ocp4-cluster/pre_infra.yml

```diff
@@ -3,7 +3,7 @@
   hosts: localhost
   connection: local
   become: false
-  gather_facts: false
+  gather_facts: true
```

Comment:
What facts are needed? Generally it is best to not gather facts then let specific roles later run the setup module to gather what they need with a test to see if needed facts were already gathered.